sdk/storage/Azure.Storage.DataMovement.Blobs/tests/BlobDestinationCheckpointDetailsTests.cs

```diff
@@ -225,10 +229,11 @@ public void Deserialize()
                 data.Serialize(stream);
                 stream.Position = 0;
                 BlobDestinationCheckpointDetails deserialized = BlobDestinationCheckpointDetails.Deserialize(stream);
-                VerifySampleValues(deserialized, DataMovementBlobConstants.DestinationCheckpointDetails.SchemaVersion);
+                VerifySampleValues_Version4(deserialized);
             }
         }
 
+        [Ignore("Renable after implementing backwards compatibility for older versions")]
```

Comment:
Will reenable this test in my next PR (issue to track: https://github.com/Azure/azure-sdk-for-net-pr/issues/2183)